Manual.md

```diff
@@ -781,6 +781,18 @@ with the character `r` in the `version` variable. The `distfiles`
 location will automatically be set as well as the package made to depend
 on `R`.
 
+- `gemspec` For packages that are built into gems using .gemspec files
+and then packaged. This build style is meant for packages that have
+C extensions and as such require patching to support cross compilation.
```

Comment:
`and as such require patching to support cross compilation.`

Is it really necessary to include this here? Sounds as if this is something the user of this build_style has to worry about the way it's currently worded.